doc/guides/kconfig/index.rst

```diff
@@ -464,6 +464,18 @@ Here are some things to check:
   they come from.
 
 
+Checking changes with :file:`scripts/kconfig/lint.py`
+*****************************************************
+
+The script :zephyr_file:`scripts/kconfig/lint.py` can be run directly to run
+various Kconfig checks. Pass ``--help`` to see options. When making Kconfig
+changes, it is a good idea to check if any related symbols show up in its
+output.
+
+Some checks are necessarily a bit heuristic, so a symbol being flagged by a
+check does not necessarily mean there's a problem.
```

Comment:
What should you do if you believe it's a false positive message?

Reply:
Not much to do at the moment unless the script is buggy, as there's no whitelist or the like. Updated it to tell people to ignore false positives.

Might add a whitelist later on.